packages/core/src/app/index.tsx

```diff
@@ -15,9 +15,6 @@ const App: React.FunctionComponent<Props> = ({ store, helmetContext = {} }) => {
         {Object.entries(store.roots).map(([namespace, Root]) => (
           <Root key={namespace} />
         ))}
-        {Object.entries(store.fills).map(([namespace, Fill]) => (
-          <Fill key={namespace} />
-        ))}
```

Comment:
So, the `Fill` components were not supposed to be placed there in the first place?

Reply:
They were. It was the first approach to integrate Slot and Fill and the way we used it in the old framework.

But then I decided to use `state.fills` and `libraries.fills` instead, which makes much more sense for Frontity.

It is explained in https://community.frontity.org/t/slot-and-fill/895